src/xdocs/config_naming.xml

```diff
@@ -690,6 +690,85 @@
       </subsection>
     </section>
 
+    <section name="LambdaParameterName">
+      <p>Since Checkstyle 8.11</p>
+      <subsection name="Description">
+        <p>
+          Check to verify lambda parameter names.
+        </p>
+      </subsection>
+
+      <subsection name="Properties">
+        <table>
+          <tr>
+            <th>name</th>
+            <th>description</th>
+            <th>type</th>
+            <th>default value</th>
+            <th>since</th>
+          </tr>
+          <tr>
+            <td>format</td>
+            <td>Specifies valid identifiers.</td>
+            <td><a href="property_types.html#regexp">Regular Expression</a></td>
+            <td><code>"^[a-z][a-zA-Z0-9]*$"</code></td>
+            <td>8.11</td>
+          </tr>
+        </table>
+      </subsection>
+
+      <subsection name="Examples">
+        <p>
+          An example of how to configure the check is:
+        </p>
+
+        <source>
+          &lt;module name="LambdaParameterName"/&gt;
+        </source>
+
+        <p>
+          An example of how to configure the check for names that begin with a lower case letter,
+          followed by letters is:
+        </p>
+        <source>
+          &lt;module name="TypeName"&gt;
```

Comment:
fix module name.

Please provide code example to show user cases where violation will(will not) be with this config.
Just place `// violation` on line.